js-conditions-loops/conditionsAndloops.js

```diff
@@ -0,0 +1,117 @@
+import chalk from "chalk";
+
+let employeeYearsOfExperience = 4;
+let employeeMonthsOfExperience = 8;
+let employeeTotalExperienceInMonths = employeeYearsOfExperience * 12 + employeeMonthsOfExperience;
+let employeeSalary = 0;
+let employeeBonus = 0;
+
+if (employeeTotalExperienceInMonths >= 6) {
+  if (employeeTotalExperienceInMonths < 12) {
+    employeeSalary = 100;
+    employeeBonus = employeeSalary * 0.01;
+  } else if (employeeTotalExperienceInMonths < 24) {
+    employeeSalary = 333;
+    employeeBonus = employeeSalary * 0.03;
+  } else if (employeeTotalExperienceInMonths < 48) {
+    employeeSalary = 555;
+    employeeBonus = employeeSalary * 0.04;
+  } else {
+    employeeSalary = 666;
+    employeeBonus = employeeTotalExperienceInMonths/12/100 * employeeSalary  ;
+  }
+}
```

Comment:
This code works, but it's hard to read. you can simplify. 
```suggestion
let employeeExperience = 0.5;// or full month format 12 month = 1 year
let employeeSalary;
let employeeBonus;


if (employeeExperience < 0.5) { // if you're gonna use full month format then here should be 6 
    employeeSalary = 0;
    employeeBonus = 0;
} else if (employeeExperience >= 0.5 && employeeExperience < 1) {// if you're gonna use full month format then here should be 6 and 12
    employeeSalary = 100;
    employeeBonus = employeeSalary * 0.01;
}
..... 
```

Reply:
fix this, please check new commit